src/content/basics/nodepools/index.md

```diff
@@ -9,6 +9,8 @@ user_questions:
   - What are node pools?
   - In which cloud environments are node pools supported?
```

Comment:
I'm not blocking this, just wondering whether this should be just firecracker, since we also support nodepools in azure?

Reply:
We discussed this in SIG product and Alex and Marta decided that Firecracker will own it anyway.